scripts/base/protocols/ssl/main.zeek

```diff
@@ -282,6 +282,11 @@ event ssl_client_hello(c: connection, version: count, record_version: count, pos
 		c$ssl$session_id = bytestring_to_hexstr(session_id);
 		c$ssl$client_ticket_empty_session_seen = F;
 		}
+
+	# add manually for SSLv2, since the handshake_message event is not raised, as there is no handshake protocol.
```

Comment:
```suggestion
	# add client hello letter for SSLv2 to history, since the handshake_message event is not raised, as there is no handshake protocol.
```

Total nit, didn't immediately know what "c" stands for :-) similarly for "s" below. Take or reject :-)